pretext/cli.py

```diff
@@ -341,7 +342,14 @@ def init(refresh):
     type=click.Choice(ASSETS, case_sensitive=False),
     help="Generates assets for target.  -g [asset] will generate the specific assets given.",
 )
-def build(target, clean, generate):
+@click.option(
+    "-P",
```

Comment:
CLI convention is to use lowercase flags

Reply:
Sure, I'll switch it. I put it as a capital because I thought it was an "advanced" option, that most users wouldn't use, so we could save lower-case letters for common options.